### PR TITLE
Adding previous period count

### PR DESCRIPTION
**Pull request by alberttoledo**
@will-horning

### CODE REVIEW

File: capitolwords_ng/capitolweb/cwapi/views.py [hunk 1]
```diff
@@ -191,10 +191,16 @@ def count_of_term_in_content(request, term):
     date_filter = {'range':{'date_issued':{'gte': start_date, 'lte':end_date}}}
     count = search.query(query).filter(date_filter).count()
 
+    previous_period_start_date = start_date + datetime.timedelta(days=0-delta)
+    previous_period_end_date = end_date + datetime.timedelta(days=0-delta)
```
Comment (will-horning) [inline capitolwords_ng/capitolweb/cwapi/views.py]: In Python you can negate a variable:
```python
0 - delta == -delta
```

**Comment by will-horning** [comment]
Other than the nit, LGTM.

**Comment by will-horning** [comment]
Derp, clicked the wrong button : )

File: capitolwords_ng/capitolweb/cwapi/views.py (hunk 1 above)
Comment (alberttoledo) [reply capitolwords_ng/capitolweb/cwapi/views.py]: 👍 done

**Pull request merged**